app/.env.dist

```diff
@@ -8,3 +8,10 @@ APP_DEBUG=1
 APP_SECRET=a276a63ddb6b5c62e994a74e2eb3fa64
 API_BASE_URI=http://15-puzzle-go:2000
 ###< symfony/framework-bundle ###
+
+###> doctrine/doctrine-bundle ###
+# Format described at http://docs.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/configuration.html#connecting-using-a-url
+# For an SQLite database, use: "sqlite:///%kernel.project_dir%/var/data.db"
+# Set "serverVersion" to your server version to avoid edge-case exceptions and extra database calls
+DATABASE_URL="pgsql://postgres@127.0.0.1:5432"
```

Comment:
As you run postgreSQL in docker, DATABASE_URL should be `pgsql://postgres@postgres:5432`. Not `127.0.0.1` anyway. And you certainly have to give it a password too.